Core/include/Acts/Utilities/HelpersMatrixGenerated.ipp

```diff
@@ -0,0 +1,825 @@
+//////////////////////////////////////////////////
+// This function is AUTOGENERATED. Do not edit! //
+//////////////////////////////////////////////////
+inline ActsMatrix<8, 8> multiply(const ActsMatrix<8, 1>& A,
+                                 const ActsMatrix<1, 8>& B) {
```

Comment:
You may want to take matrices A and B by value. The compiler should be able to elide the copy, and this will eliminate any risk of losing compiler optimizations due to pointer aliasing issues.

Reply:
This has a negative effect. I'm guessing the copy elision fails.